custom_components/renault/translations/en.json

```diff
@@ -4,8 +4,7 @@
       "kamereon_no_account": "Unable to find Kamereon account."
     },
     "error": {
-      "invalid_credentials": "Invalid credentials.",
-      "preload_failed": "Unable to preload Renault keys."
+      "invalid_credentials": "Invalid credentials."
```

Comment:
I'm not sure but perhaps we should use the common strings here To reuse the system translation already done. No ?
```
"invalid_credentials": "[%key:common::config_flow::error::invalid_auth%]"
```

Reply:
I think the common strings are generated by HA core during the "publishing" time, so they cannot be used inside custom components.